src/test/java/fi/fmi/avi/converter/json/JSONSpaceWeatherAdvisoryAmd82ConverterTest.java

```diff
@@ -123,25 +125,29 @@ public void testSWXSerialization() throws Exception {
         om.registerModule(new Jdk8Module());
         om.registerModule(new JavaTimeModule());
 
-        final InputStream is = JSONSigmetConverterTest.class.getResourceAsStream("swx1.json");
+        final InputStream is = JSONSigmetConverterTest.class.getResourceAsStream("swx-amd82.json");
         Objects.requireNonNull(is);
 
         final String reference = IOUtils.toString(is, "UTF-8");
 
-        final SpaceWeatherAdvisoryAmd82Impl SWXObject = SpaceWeatherAdvisoryAmd82Impl.builder()
+        final SpaceWeatherAdvisoryAmd82Impl advisory = SpaceWeatherAdvisoryAmd82Impl.builder()
                 .setIssuingCenter(getIssuingCenter())
                 .setIssueTime(PartialOrCompleteTimeInstant.builder().setCompleteTime(ZonedDateTime.parse("2020-02-27T01:00Z[UTC]")).build())
                 .addAllAnalyses(getAnalyses(false))
                 .setPermissibleUsageReason(AviationCodeListUser.PermissibleUsageReason.TEST)
                 .addAllPhenomena(Arrays.asList(SpaceWeatherPhenomenon.fromWMOCodeListValue("http://codes.wmo.int/49-2/SpaceWxPhenomena/HF_COM_MOD"),
                         SpaceWeatherPhenomenon.fromWMOCodeListValue("http://codes.wmo.int/49-2/SpaceWxPhenomena/GNSS_MOD")))
-                .setAdvisoryNumber(getAdvisoryNumber())
-                .setReplaceAdvisoryNumber(Optional.empty())
+                .setAdvisoryNumber(advisoryNumber(2020, 1))
+                .addAllReplaceAdvisoryNumber(ImmutableList.of(
+                        advisoryNumber(2019, 40),
+                        advisoryNumber(2019, 41),
+                        advisoryNumber(2019, 42))
+                )
```

Comment:
I would be cautious using Guava within this library, to minimize dependencies. Though, this is only test code. But actually, we have a native way of avoiding `ImmutableList` here (and similar cases):

```suggestion
                .addReplaceAdvisoryNumber(
                        advisoryNumber(2019, 40),
                        advisoryNumber(2019, 41),
                        advisoryNumber(2019, 42))
```